source/_components/history.markdown

```diff
@@ -18,8 +18,33 @@ The `history` component will track everything that is going on within Home Assis
 To enable the history option in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yaml entry
+# Example configuration.yaml entry without any filters
```

Comment:
This should only be a basic minimal config. Remove the "without any filters" part. This is for consistency in documentation. Explain more in the others examples instead.